### PR TITLE
Bump down headings in markdown content

### PR DESCRIPTION
**Pull request by IanMadd**
Signed-off-by: IanMadd <imaddaus@chef.io>

This drops all the headings in the MD files by one level, H1 --> H2, H2 --> H3, etc...

## Description
Each page should have only one H1 heading which is the page title. This means that the headings in the text should all be H2 or below.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have run the pre-merge tests locally and they pass.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [X] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: www/content/workstation/chef_shell.md [hunk 1]
```diff
@@ -19,12 +19,12 @@ aliases = ["/chef_shell.html", "/chef_shell/"]
 The chef-shell executable is run as a command-line tool.
 
 Modes
-=====
+-----
```
Comment (kagarmoe) [inline www/content/workstation/chef_shell.md:22]: Why are we using .rst-style sections?

Comment (IanMadd) [reply www/content/workstation/chef_shell.md:22]: Legacy and they're commonmark compliant. No reason other than that.

**Comment by tyler-ball** [comment]
@IanMadd I browsed through a few pages and they look good, but the TOC on the right sidebar has an extra bullet point:
image

I don't see that when looking at the [live version](https://docs.chef.io/workstation/chef_run/)

**Comment by IanMadd** [comment]
@tyler-ball That's fine. It's pulling the theme from chef-web-docs. The TOC there is still configured for H1 through H3 headings, but as soon as chef/chef-web-docs#2443 is merged then the extra bullet point will go away.

**Pull request merged**